ui/lib/pki/addon/components/pki-issuer-cross-sign.js

```diff
@@ -210,6 +228,16 @@ export default class PkiIssuerCrossSign extends Component {
   @action
   reset() {
     this.signedIssuers = [];
+    this.validationErrors = [];
     this.formData = [];
   }
+
+  nameValidation(nameInput, existing) {
```

Comment:
the intent here was to mirror how the model validations decorator returns error messages